src/views/Leads/index.js

```diff
@@ -209,7 +229,7 @@ export default class Leads extends React.PureComponent {
                 sortable: true,
                 order: 9,
                 modifier: row => (
-                    <div className={styles.status}>
+                    <div className="status">
```

Comment:
Why is a non-scoped className used here?

Reply:
The styles for Tables are stored in separate file (inside Tables/styles.scss) so non-scoped className.